kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -379,16 +379,21 @@
       height: $height-of-prompt + $height-of-checkbox + $height-of-error
       margin-bottom: $form-item-spacing // margin defined for k-textbox
       margin-top: $k-textbox-text-distance
+      max-width: 100%
       &-view-prompt
 
         // duplicating styles from `<a>` in core theme
         color: $core-action-normal
         transition: color $core-time ease-out
+        max-width: 100%
+        white-space: nowrap
+        overflow: hidden
+        text-overflow: ellipsis
```

Comment:
why is this preferable to just wrapping the line?

Reply:
Also not sure about this ^